source/process/community-guidelines.rst

```diff
@@ -0,0 +1,127 @@
+============================================================
+Mattermost Community Guidelines
+============================================================
+
+This document provides easy-to-follow community guidelines for:
+
+- `GitHub Help Wanted Issues in Platform Repository`_
+- `Mattermost Community Forums`_
+
+----
+
+GitHub Help Wanted Issues in Platform Repository
+---------------------------------------------------------
+
+Goal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Increase Mattermost usage through larger and more numerous deployments via a vibrant open source community that contributes features that would not otherwise be offered.
+
+Principles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Contributors get a worthwhile benefit from their contributions, from the feature directly and/or from social recognition
+- Community members should have a low barrier to contribute
+- Community development workflow should be clear, efficient and effective
+
+Creating Help Wanted issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To create a Help Wanted issue in the `platform repository <https://github.com/mattermost/platform>`_, follow these steps:
+
+1 - Create a ticket in Jira that is well-defined and unambiguous. 
+
+The issue should be written with the mindset that the contributor might have no or limited experience with the Mattermost code base and limited exposure to the Mattermost product.
+
+  .. note::
+    Below are a few reasons why Jira ticket for Help Wanted issues is recommended:
+    
+      1. Jira can be used to prioritize Help Wanted issues internally and are easily searchable by Mattermost staff, community, and customers
+      2. Each Jira ticket goes through the  `triage meeting <https://docs.mattermost.com/process/training.html#triage-meeting>`_ for dev and PM approval
+      3. Zapier integration automatically creates GitHub Help Wanted issues from labelled Jira tickets, requiring no additional mana
+      4. Resolved Jira tickets are automatically assigned to a QA, who tests them agains the ``master`` branch.
+
+2 - After creating the ticket, add "(Proposed APR)" to its title, so the triage team knows to consider it for a ``help-wanted`` label.
+
+3 - If the Jira ticket is accepted as a Help Wanted issue during `triage <https://docs.mattermost.com/process/training.html#triage-meeting>`_, a ``help-wanted`` label is added. This action automatically triggers a zap that creates a new issue in the `platform repository <https://github.com/mattermost/platform>`_.
```

Comment:
Should we add a note about what we should look for in triage to accept a Help Wanted? (Also should we assign difficulty level/programming language then? Or afterwards?)